src/components/modals/FlipInputModal.js

```diff
@@ -92,6 +91,19 @@ class FlipInputModalComponent extends React.PureComponent<Props, State> {
 
   handleSendMaxAmount = () => this.props.updateMaxSpend(this.props.walletId, this.props.currencyCode)
 
+  renderTitle = () => {
+    const styles = getStyles(this.props.theme)
+    return (
+      <View style={styles.headerContainer}>
```

Comment:
You should try using `justifyContent: "space-between"` on this container component, so you can get rid of the `styles.spacer` view.